generator-honeycomb/app/templates/.config/_webpack.config.js

```diff
@@ -26,15 +26,15 @@ const defaultConfig = {
     rules: [
       {
         test: /\.js$/,
-        loader: 'babel!eslint',
+        loader: 'babel-loader!eslint-loader',
```

Comment:
wouldn´t it be nicer to use:

```javascript
resolveLoader: {
    moduleExtensions: ['-loader'],
  },
```

instead of writing loader everywhere?